### DISCUSSION TITLE
Modified the configuration (creation form) of task in project.

### DISCUSSION THREAD

**Discussion by paljitender** [exoplatform/exo-community#56] [Q&A]
Requirement is to create more number of fields in the task/challenges in the project.

In existing configuration we have only 4 fields available in task. Wanted to increase the fields upto 30. 

Please help in this.

**Comment by rdenarie** [comment]
Hi @paljitender 

The possibility to customize tasks fields is not available currently.

I encourage you to post a Feature Request issue in Meeds Project : https://github.com/Meeds-io/meeds/issues/new?assignees=&labels=&projects=&template=feature_request.md&title=

**Reply by paljitender** [reply]
Thanks @rdenarie,

So customization is not possible iin exo at form level like, when we create any entity (Space, Community, project)..

**Reply by rdenarie** [reply]
I confirm that there is no customization possible in such entity.

Some are configurable, like the user profile
An administrator can  configure and add fields the user can fill in his profile.

Can you briefly describe functionnaly what you want to do exactly ?
